### DISCUSSION TITLE
Should `is-a.dev` be moved to cloudflare?

### DISCUSSION THREAD

**Discussion by mtgsquad** [is-a-dev/register#1112] [Discussion] [answered]
It would be a painful thing to do, but we would be happy to know your thoughts on this. Please let us know about your thoughs on migrating from Namecheap DNS to Cloudflare DNS.

> Related: #1091

**Comment by Satindar31** [comment]
I Agree

**Comment by ServerDeveloper9447** [comment]
I agree

**Comment by ghost** [comment]
I agree

**Comment by phenax** [comment, marked as answer]
This discussion skipped my mind completely. As discussed with the other maintainers, we cannot move our current setup to cloudflare due to a hard limit on the number of dns records. We are however looking for higher level alternatives for the future if things need to scale up further.

**Reply by 0vol** [reply]
i disogree
